### PR TITLE
Group actions

### PR DESCRIPTION
**Pull request by kwm4385**
Adds leave group and EP editing functionality to the group page.

image

Users can be selected and moved back and forth using the buttons. There is a confirmation dialog for leaving the group.

### CODE REVIEW

**Comment by kwm4385** [comment]
I'll be circling back on the page to finalize styling and usability, but want to get the raw functionality in first.

File: __tests__/__snapshots__/group_spec.js.snap [hunk 1]
```diff
@@ -34,37 +41,58 @@ exports[`GroupView renders GroupView 1`] = `
...
+      className="five wide computer sixteen wide mobile column"
     >
       <h3>
         Escalation Order:
       </h3>
       <a
-        className="arrow_box"
-        href="/user/58a09497b73f14c7d6d64437/"
+        className="box arrow_box"
+        onClick={[Function]}
```
Comment (paradime) [inline __tests__/__snapshots__/group_spec.js.snap]: Where is this function defined?

File: src/routes/Group/components/GroupView.js [hunk 2]
```diff
@@ -48,18 +119,52 @@ class GroupView extends React.Component {
...
         <Divider/>
         <Grid>
-          <Grid.Column mobile={16} computer={6}>
+          <Grid.Column mobile={16} computer={5}>
             <h3>Escalation Order:</h3>
             {this.active()}
           </Grid.Column>
-          <Grid.Column mobile={16} computer={6}>
+          <Grid.Column verticalAlign="middle" mobile={16} computer={1}>
+            <Button
```
Comment (paradime) [inline src/routes/Group/components/GroupView.js]: Does it make sense to move this into it's own method?

File: __tests__/__snapshots__/group_spec.js.snap (hunk 1 above)
Comment (kwm4385) [inline __tests__/__snapshots__/group_spec.js.snap]: This was generated by the tests.

**Pull request merged**